src/test/java/org/gridsuite/study/server/service/client/AbstractRestClientTest.java

```diff
@@ -1,74 +0,0 @@
-/*
- * Copyright (c) 2022, RTE (http://www.rte-france.com)
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/.
- */
-
-package org.gridsuite.study.server.service.client;
-
-import okhttp3.HttpUrl;
-import okhttp3.mockwebserver.Dispatcher;
-import okhttp3.mockwebserver.MockWebServer;
-import org.gridsuite.study.server.ContextConfigurationWithTestChannel;
-import org.gridsuite.study.server.utils.elasticsearch.DisableElasticsearch;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.runner.RunWith;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.test.context.junit4.SpringRunner;
-
-import java.io.IOException;
-
-/**
- * @author Thang PHAM <quyet-thang.pham at rte-france.com>
- */
-@RunWith(SpringRunner.class)
-@SpringBootTest
-@DisableElasticsearch
-@ContextConfigurationWithTestChannel
-public abstract class AbstractRestClientTest {
```

Comment:
Maybe don't delete it

Reply:
answered at https://github.com/gridsuite/study-server/pull/626#discussion_r1812929711